PackageExplorer/MainWindow.xaml.cs

```diff
@@ -177,6 +184,63 @@ private bool OpenLocalPackageCore(string packagePath)
             return false;
         }
 
+        private void InitPackageFileWatcher(string packagePath)
+        {
+            if (_packageFileWatcher == null)
+            {
+                _packageFileWatcher = new FileSystemWatcher();
+                _packageFileWatcher.IncludeSubdirectories = false;
+                _packageFileWatcher.Changed += OnFileChanged;
+                _packageFileWatcher.Deleted += OnFileChanged;
+                _packageFileWatcher.Renamed += OnFileChanged;
+            }
+            _packageFileWatcher.Path = Path.GetDirectoryName(packagePath);
+            _packageFileWatcher.Filter = Path.GetFileName(packagePath);
+            _packageFileWatcher.EnableRaisingEvents = true;
+        }
+
+        protected override void OnActivated(EventArgs e)
+        {
+            CheckFileChanges();
+        }
+
+        private async void CheckFileChanges()
+        {
+            if (!_packageFileChanged) return;
+            _packageFileChanged = false;
+
+            if (_packageFileWatcher != null)
+            {
+                var packagePath = Path.Combine(_packageFileWatcher.Path, _packageFileWatcher.Filter);
+                if (File.Exists(packagePath))
+                {
+                    UIServices.Show("Package file has been changed", MessageLevel.Information);
```

Comment:
Will put the strings into resources. Are they ok?

Reply:
What is the purpose of notifying the user it has been changed? Instead of a popup, can we alter the title or status bars?

Reply:
Just to notify the user but the view model can be disposed or reloaded without a message.
What do you mean with updating the title? The view model is in an invalid state if the file is renamed / deleted / modified.

Reply:
what if we copy the file to a temp directory upon open, so it can never be renamed. We'd preserve the original path for save operations.

Reply:
Ah yes, that would also work, but I wanted to do it the hard way 😄 
Will redo it.